### PR TITLE
Solving bug #125

### PR DESCRIPTION
**Pull request by Fenicio**

### CODE REVIEW

File: src/scripts/Select.js [hunk 1]
```diff
@@ -59,5 +59,5 @@ Select.propTypes = {
 Select.isFormElement = true;
 
 export const Option = props => (
-  <option { ...props } />
+  <option { ...props } >{ props.label ? props.label || "" }</option>
```
Comment (stomita) [inline src/scripts/Select.js]: If you want `Option` to accept `label` property, it should be
```javascript
export const Option = props => {
  const { label, children, ...pprops } = props;
  return <option { ...pprops }>{ label || children }</option>;
}
```

Also it needs propTypes to pass the lint.

**Comment by bondz** [comment]
You need to add `PropTypes` validation, check CI or run `npm test` in your cli

**Comment by bondz** [comment]
@fenicio looks like you got everything sorted out. You just need to update the `snapshots (storyshots)` to appease CI. Sorry for the frustration, your commit messages though :D

**Comment by Fenicio** [comment]
So I learned a lot while doing this, as to how to correctly code React and it's testing ecosystem, but I still have a doubt.

The lint is finally passing, the thingy is working. Am I expected to update the .storybook/*.shot ?

**Comment by bondz** [comment]
@Fenicio yes, but not manually. If you're ok with the changes, you can run

```bash
npm run test:storyshots -- -u
```

Or If you're using Windows.

```cmd
set NODE_ENV=test
.\node_modules\.bin\storyshots -u
```

That should fix things.

**Comment by stomita** [comment]
@Fenicio Sorry for frustrating. the storybook spot generation might have differs in environment. I'll pick this PR and merge it with removing errors.

**Comment by Fenicio** [comment]
Thank you :)

**Pull request merged**